base/pkg/generate.jl

```diff
@@ -149,6 +150,17 @@ function travis(pkg::AbstractString; force::Bool=false)
         #script:
```

Comment:
To clear things up, and since people switching over to the default script are mistakenly thinking that `coverage=true` is set, how about adding this line here for now?: `# (the following snippet differs from the default test script in that coverage/Coveralls are not enabled by default)`